tests/unit/test_specification.py

```diff
@@ -103,6 +103,11 @@ def test_field_typology():
     assert specification.field_typology("field-categories") == "category"
 
 
+def test_field_parent():
```

Comment:
might as well add a test for a missing parent-field returning "" or None (depending on outcome of previous comment).

Reply:
If the specification csv is loaded the `parent_field` will always exist so we decided a test for returning None isn't necessary.